app/poros/search.rb

```diff
@@ -0,0 +1,16 @@
+class Search
```

Comment:
I like that you made one presentation object to pass to the view. This is a good pattern.